### PR TITLE
Clip-paths for headings

### LINKED ISSUE #31: Style H2 and H3 headers
I'm thinking something like this that helps it stand out on the page:

Screen Shot 2019-06-28 at 7 58 29 PM

Of course, with the random polygon effect, and rotation something like from -2 to +2 degrees at random, and the amount it moves out to the left from .5-1.25em.

Sound good?

I'm not 100% on the colors yet, but we can do this for now with h2.

For H3 maybe something similar but smaller and only spanning half the page for h3

H4 we can try being more tame with. 

Will also need to figure out what to do when these happen within certian blocks like Media & Text. But we can solve that when we come across it.

Here's the code I used for the proof of concept above:
```
padding: 1em;
margin-left: -1em;
transform: rotate(-1.5deg);
background: #968c83;
color: #e8e6df;
```

### PR DESCRIPTION
**Pull request by zoracreates**
Randomized the clip-paths and rotations for headings. This should close #31.

### CODE REVIEW

File: css/00-settings.css [hunk 1]
```diff
@@ -6,6 +6,21 @@
...
+	--topLeftX: 0;
+	--topLeftY: 0;
+	--topRightX: 0;
+	--topRightY: 0;
+	--bottomRightX: 0;
+	--bottomRightY: 0;
+	--bottomLeftX: 0;
+	--bottomLeftY: 0;
+	--rotate: 0;
+	--topPadding: 1rem;
```
Comment (slambert) [inline css/00-settings.css]: Oooh boy this is exciting. 

A suggestion - can this be:

	--topPadding: 2rem;
	--bottomPadding: 2rem;
	--leftPadding:2rem;
	--rightPadding: 2rem;

It's feels less cramped.

Comment (slambert) [reply css/00-settings.css]: Also, can we slide these over a bit. In my quick look I couldn't figure out how to code a demo of what I mean, but I'd like them to drift to the left a bit. Partly so the edge of the letters in the headers and paragraphs seem to line up, or if the header goes further to the left works to interrupt the flow and catch the eye which, even if breaking a text layout rule, is in the spirit of what headers do. (Am I over explaining?)

Screen Shot 2019-08-11 at 8 09 53 PM

Comment (slambert) [reply css/00-settings.css]: For example, I did get this working and I love how it turned out:

Screen Shot 2019-08-11 at 8 18 49 PM

I just added

	margin-left: -4em;

To line 16 of c4aa-clippaths.css

Comment (laras126) [reply css/00-settings.css]: @slambert is this ready to merge then?

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const headings = document.querySelectorAll( 'h2, h3, h4' );
```
Comment (laras126) [inline js/script.js]: Ehhh....this seems dangerous to me to have it be so global. My inclination would be to add a body class similar to c4aa-duotone and the grid and apply to headings within `.c4aa-clipPaths` - that way the file naming matches up as well. What do you think @zoracreates?

File: css/00-settings.css (hunk 1 above)
Comment (slambert) [reply css/00-settings.css]: Nope nope! Above are just suggestions/examples.

File: js/script.js (hunk 2 above)
Comment (laras126) [reply js/script.js]: Going to merge and we can update if it becomes an issue!!

Comment (zoracreates) [reply js/script.js]: Sounds good. Thanks! I didn't get a chance to work on this earlier.

**Pull request merged**